spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RetryingMessageListenerAdapter.java

```diff
@@ -37,7 +40,7 @@
  */
 public class RetryingMessageListenerAdapter<K, V>
```

Comment:
Well, there is : http://docs.spring.io/spring-kafka/docs/2.0.0.BUILD-SNAPSHOT/reference/html/_reference.html#_retrying_deliveries, but not so much:

> A retry adapter is not provided for any of the batch message listeners.

Would be great to elaborate.
Thanks